{
  "affected_release": null,
  "package_state": [
    {
      "product_name": "Red Hat Enterprise Linux 7",
      "fix_state": "Affected",
      "package_name": "mariadb",
      "cpe": "cpe:/o:redhat:enterprise_linux:7"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "mariadb:10.3/mariadb",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 8",
      "fix_state": "Affected",
      "package_name": "mariadb:10.5/mariadb",
      "cpe": "cpe:/o:redhat:enterprise_linux:8"
    },
    {
      "product_name": "Red Hat Enterprise Linux 9",
      "fix_state": "Affected",
      "package_name": "mariadb",
      "cpe": "cpe:/o:redhat:enterprise_linux:9"
    },
    {
      "product_name": "Red Hat Software Collections",
      "fix_state": "Affected",
      "package_name": "rh-mariadb105-mariadb",
      "cpe": "cpe:/a:redhat:rhel_software_collections:3"
    }
  ],
  "threat_severity": "Important",
  "public_date": "2023-09-20T00:00:00Z",
  "bugzilla": {
    "description": "node crashes with Transport endpoint is not connected mysqld got signal 6",
    "id": "2240246",
    "url": "https://bugzilla.redhat.com/show_bug.cgi?id=2240246"
  },
  "cvss": {
    "cvss_base_score": "",
    "cvss_scoring_vector": "",
    "status": ""
  },
  "cvss3": {
    "cvss3_base_score": "7.5",
    "cvss3_scoring_vector": "CVSS:3.1/AV:N/AC:L/PR:N/UI:N/S:U/C:N/I:N/A:H",
    "status": "draft"
  },
  "iava": "",
  "cwe": "CWE-400",
  "statement": "",
  "acknowledgement": "",
  "name": "CVE-2023-5157",
  "document_distribution": "",
  "details": [
    "No description is available for this CVE."
  ],
  "references": [
    "https://www.cve.org/CVERecord?id=CVE-2023-5157\nhttps://nvd.nist.gov/vuln/detail/CVE-2023-5157"
  ]
}